shared/libraries/websocket_streaming/tests/test_websocket_client_device.cpp

```diff
@@ -106,14 +107,15 @@ TEST_F(WebsocketClientDeviceTest, SingleSignalWithDomain)
 
     std::this_thread::sleep_for(std::chrono::milliseconds(250));
 
-    server->broadcastPacket(signalId, PropertyChangedEventPacket("Name", "NewName"));
-    server->broadcastPacket(signalId, PropertyChangedEventPacket("Description", "NewDescription"));
+    testSignal.asPtr<IComponentPrivate>().unlockAllAttributes();
+    testSignal.setName("NewName");
+    testSignal.setDescription("NewDescription");
     std::this_thread::sleep_for(std::chrono::milliseconds(250));
 
     ASSERT_EQ(clientDevice.getSignals()[0].getName(), "NewName");
     ASSERT_EQ(clientDevice.getSignals()[0].getDescription(), "NewDescription");
 
-    ASSERT_THROW(clientDevice.getSignals()[0].setName("ClientName"), AccessDeniedException);
+    ASSERT_NO_THROW(clientDevice.getSignals()[0].setName("ClientName"));
```

Comment:
Shouldn't the use of `setName`/`setDescription` methods be restricted for mirrored signals, at least for pseudo device signals? 
On the flip side, the drawback of such approach would be the need for some alternative to the protected property interface, which was previously used to initialize/update `name` and `description` with values received via the streaming protocol.

Reply:
They are restricted in the sense that they do not do anything. The setters such as `setName / Description` do not throw an exception, but instead simply return `OPENDAQ_IGNORED`. By default the mirrored signals should have these fields locked.